test/ProcessTest.php

```diff
@@ -67,7 +67,7 @@ public function testCommandCanRun() {
 
     public function testCommandArray() {
         Loop::run(function () {
-            $process = new Process([self::CMD_PROCESS]);
+            $process = new Process(['echo', 'foo']);
```

Comment:
This should actually be applied everywhere. An array is used to build the command from its arguments, but `self::CMD_PROCESS` is already a full command string.